Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openshift/origin-base
+FROM openshift/origin-release:golang-1.10 AS builder
 
 ENV GOPATH="/go"
```

Comment:
Actually GOPATH in the builder includes /go and an SCL-specific path. I'm not certain what the consequences are of leaving that out. Thus I would suggest not setting this. I would use another variable or just "/go" (which is shorter anyway...)